frontend/src/lib/components/LemonCalendar/LemonCalendar.scss

```diff
@@ -0,0 +1,19 @@
+.LemonCalendar {
+    display: flex;
```

Comment:
a lot of this could be utility classes on the thing itself

Reply:
moved over as much as I could